test/react/server_rendering/webpacker_containers_test.rb

```diff
@@ -11,6 +11,8 @@ def test_it_loads_JS_from_the_webpacker_container
       WebpackerHelpers.compile
       container = React::ServerRendering::WebpackerManifestContainer.new
       assert_not_empty container.find_asset('application.js')
+      container = React::ServerRendering::SeparateServerBundleContainer.new
+      assert_not_empty container.find_asset('server_rendering.js')
```

Comment:
How does this test work?

Reply:
It really doesn't do much more than confirm that `React::ServerRendering::SeparateServerBundleContainer.new.find_asset('server_rendering.js')` doesn't throw an error.

But that's the same level of testing for `React::ServerRendering::WebpackerManifestContainer` that was already there so I'm simply applying the same standards.